package.json

```diff
@@ -69,6 +69,7 @@
     "@types/node": "13.1.8",
     "@types/sinon": "9.0.5",
     "@types/ws": "7.2.0",
+    "@types/mime": "2.0.3",
```

Comment:
Let's do a separate PR for this since we should also update our `package-lock.json`.